README.md

```diff
@@ -1,26 +1,86 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+# Airbyte K8s Operator
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+This is the Kubernetes Python Operator for [Airbyte](https://airbyte.com/).
 
-Use links instead.
--->
+## Description
 
-# airbyte-k8s-operator
+Airbyte is an open-source data integration platform designed to centralize and
+streamline the process of extracting and loading data from various sources into
+data warehouses, lakes, or other destinations.
 
-Charmhub package name: operator-template
-More information: https://charmhub.io/airbyte-k8s-operator
+This operator provides an Airbyte server, and consists of Python scripts which
+wraps the versions distributed by
+[Airbyte](https://hub.docker.com/r/airbyte/server).
 
-Describe your charm in one or two sentences.
+## Usage
 
-## Other resources
+Note: This operator requires the use of juju>=3.1.
 
-<!-- If your charm is documented somewhere else other than Charmhub, provide a link separately. -->
+### Deploying PostgreSQL Database
 
-- [Read more](https://example.com)
+The Airbyte and PostgreSQL operators can be deployed and connected to each
+other using the Juju command line as follows:
 
-- [Contributing](CONTRIBUTING.md) <!-- or link to other contribution documentation -->
+```bash
+juju deploy airbyte-k8s --trust
+juju deploy postgresql-k8s --channel 14/edge --trust
+juju relate airbyte-k8s postgresql-k8s
+```
 
-- See the [Juju SDK documentation](https://juju.is/docs/sdk) for more information about developing and improving charms.
+### Deploying Minio
+
+The Airbyte and Minio operators can be deployed and connected to each
```

Comment:
For each section it would be good to have what the role is of the supporting charm, aka why do we connect Minio.